clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -1,8 +1,42 @@
 use clar2wasm::tools::crosscheck;
+use clarity::vm::Value;
+use proptest::prelude::prop;
 use proptest::proptest;
+use proptest::strategy::Strategy;
 
 use crate::PropValue;
 
+fn begin_strategy() -> impl Strategy<Value = (String, PropValue, bool)> {
```

Comment:
Does it make sense to have the separated `begin_strategy` here? Do you intend to reuse this method somewhere?

For now, I have the impression that the code looks like
```rust
fn _foo() {
    // super 
    // long 
    // body
}

pub fn foo() {
    _foo()
}
```

Reply:
Don't you think this strategy should be used in the body of `define-private`, `define-public` and `define-read-only`?

Reply:
I don't know, you tell me! But in this case, I would give a more generic name.

Reply:
Yeah. I'll refactor it a bit in upcoming PR. it's almost ready and waiting for this to be merged

Reply:
Okay, if this will be changed in the very next PR, no problem for me.